oem7_raw_msgs:
- 42    # BESTPOS
- 99    # BESTVEL    
- 1429  # BESTGNSSPOS    
- 1538  # PPPPOS          
- 1465  # INSPVAX                  
- 1825  # INSUPDATESTATUS
- 1462  # RAWIMUSX
- 41    # RAWEPHEM    
- 723   # GLOEPHEMERIS            
- 1696  # BDSEPHEMERIS
- 1310  # GALFNAVEPHEMERIS     
- 1309  # GALINAVEPHEMERIS
- 43    # RANGE            
- 1335  # HEADING2                
- 101   # TIME                   
- 93    # RXSTATUS
- 1945  # INSCONFIG
- 1719  # TERRASTARINFO
- 1720  # TERRASTARSTATUS
- 42274 # BYINS2